util/settings/settings.go

```diff
@@ -266,6 +270,20 @@ func (mgr *SettingsManager) GetResourceOverrides() (map[string]v1alpha1.Resource
 	return resourceOverrides, nil
 }
 
+// GetKustomizeBuildOptions loads the kustomize build options from argocd-cm ConfigMap
+func (mgr *SettingsManager) GetKustomizeBuildOptions() (string, error) {
```

Comment:
@alexmt is this the right way to do this now?

Reply:
Yes, the idea is that there should be accessor methods going forward, instead of returning an entire Settings object.